modules/metrics_calc.py

```diff
@@ -0,0 +1,30 @@
+from torch import no_grad
+from torch.utils.data import Subset, DataLoader
+from numpy.random import randint
+
+def true_accuracy(dataloader, model, device):
+    total, correct = 0, 0
+    with no_grad():
+        for images, labels in dataloader:
+            images, labels = images.to(device), labels.to(device)
+            outputs = model(images)
+    #         outputs = outputs.to(device)
+            _, predicted = outputs.data.max(dim=1)
+            total += labels.size(0)
+            correct += (predicted == labels).sum().item()
+        return correct / total
+    
+def approx_accuracy(dataset, model, device, reduce):
```

Comment:
In general, it is better to use something like `fraction` instead of `reduce` - it is more clear allows better control.
```
approx_accuracy(..., fraction=0.5):
```